NBitcoin/RPC/RPCTransactionInput.cs

```diff
@@ -0,0 +1,23 @@
+#if !NOJSONNET
+using Newtonsoft.Json;
+namespace NBitcoin.RPC
+{
+	public class RPCTransactionInput
+	{
```

Comment:
Can you add `RPCTransactionInput(TxIn input)` and call that in the `ToRPCInputs` ? This is more easy to discover.